src/app/tests/TestICDManager.cpp

```diff
@@ -136,45 +137,45 @@ class TestICDManager
         TestContext * ctx = static_cast<TestContext *>(aContext);
 
         // Setting a requirement will transition the ICD to active mode.
-        ctx->mICDManager.SetKeepActiveModeRequirements(ICDManager::KeepActiveFlags::kCommissioningWindowOpen, true);
+        GetICDNotifier()->ActiveRequest(ICDNotify::KeepActiveFlags::kCommissioningWindowOpen, true);
```

Comment:
How deep do true/false booeans go through the stack? I find their usage as arguments in multi-argument methods usually harder to follow/extend. For clarity, did we consider having pairs of start/stop or enter/leave or similar? I would read code like `ICDNotifier::GetInstance()->EnterActive(ICDNotify::KeepActiveFlags::kCommissioningWindowOpen)` and `ICDNotifier::GetInstance()->ExitActive(ICDNotify::KeepActiveFlags::kCommissioningWindowOpen)`

Reply:
The boolean goes 1 level deep, but I like the idea of an enter and exit function